Remotion/ObjectBinding/Web/UI/Controls/BocColumnDefinition.cs

```diff
@@ -53,7 +55,7 @@ public override string ToString ()
     {
       string displayName = ItemID;
       if (string.IsNullOrEmpty (displayName))
-        displayName = ColumnTitle;
+        displayName = ColumnTitle.ToString();
```

Comment:
```suggestion
        displayName = ColumnTitle.ToString(WebStringEncoding.HtmlWithTransformedLineBreaks);
```

Reply:
I'm not sure about this one. Imo it should keep non-html linebreaks.
@MichaelKetting  ?

Reply:
correct. That's the resting happy face for this sort of thing. There's probably a usecase for removing linebreaks that have been specified in the strings, but I can't think of a practical case right now.